### PR TITLE
change password button is disabled

### PR DESCRIPTION
**Pull request by Mishra121**
Fixes #[167]

#### Changes proposed in this pull request:
- disable button works properly

#### Demo Link: 
[Add here the link where your changes can be seen]

#### Screenshots for the change: 
Screenshot from 2019-05-18 16-13-57

### CODE REVIEW

**Review by samagragupta — COMMENTED** [review]
Please screenshot or gif and also issue number in the description.

**Comment by Mishra121** [comment]
pull request description updated.

**Review by samagragupta — APPROVED** [review]

File: src/scripts/options.js [hunk 1]
```diff
@@ -190,6 +189,19 @@ function login(event) {
 	});
 
 }
+
+var changepasswordsubmitButton = document.getElementById("changepasswordsubmit");
```
Comment (tstreamDOTh) [inline src/scripts/options.js]: Declare the variables at the top

**Review by tstreamDOTh — CHANGES_REQUESTED** [review]

**Comment by tstreamDOTh** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- src/scripts/options.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/tstreamDOTh/susi_firefoxbot/pullRequest?prid=5777456&bid=18914679)

**Pull request closed without merging**